packages/caliper-burrow/lib/burrowClientFactory.js

```diff
@@ -38,7 +38,7 @@ class BurrowClientFactory {
      * @returns {Object} the child process
      */
     async spawnWorker() {
-        const child = childProcess.fork(path.join(__dirname, './burrowClientWorker.js'));
+        const child = childProcess.fork(path.join(__dirname, './burrowClientWorker.js'), process.argv.slice, { env: process.env});
```

Comment:
Shouldn't the second argument be `process.argv.slice(1)`? The actual call to `slice` is missing. Same for the other client factories.